docs/source/markdown/podman-build.1.md

```diff
@@ -479,7 +479,7 @@ Windows base images, so using this option is usually unnecessary.
 
 Output destination (format: type=local,dest=path)
 
-The --output (or -o) option extends the default behavior of building a container image by allowing users to export the contents of the image as files on the local filesystem, which can be useful for generating local binaries, code generation, etc.
+The --output (or -o) option extends the default behavior of building a container image by allowing users to export the contents of the image as files on the local filesystem, which can be useful for generating local binaries, code generation, etc. (This option is not available with the remote Podman client, including Mac and Windows (excluding WSL2) machines)
```

Comment:
small nit I prefer if this would be on the next line but not worth to repush

Reply:
ah it looks like this commit is exactly picked for `4.1` will change it if i repush it again otherwise I just tried to replicate how its written for other commands but will change on another repush.